.github/workflows/ci.yml

```diff
@@ -139,6 +140,12 @@ jobs:
         if: ${{ matrix.os == 'windows-2022' }}
         run: |
           Set-MpPreference -DisableRealtimeMonitoring $true
+      - name: Set new git config - Windows
+        if: ${{ matrix.os == 'windows-2022' }}
+        run: |
+          git config --global pack.window 1
+          git config --global core.compression 0
+          git config --global http.postBuffer 524288000
```

Comment:
Why only for `build-tools` and only for windows?

Reply:
That was my fault, we should have that in all windows matrix usage, let me fix